src/components/search/SearchField.js

```diff
@@ -12,14 +12,29 @@ const SearchField = ({ value, toggle, setToggle, fieldRef, createModal }) => {
   //state variables
   const [options, setOptions] = useState([]);
   const [list, setList] = useState([]);
+  const [filtering, setFiltering] = useState(true);
+
+  // sections of the dropdown
+  const listSections = [
```

Comment:
let's either find a way to lift this, or get it wrapped in useMemo.

Reply:
Thanks for catching that. I removed the filtering state and did a simple check in the filterData function. First time using useMemo but I can see why you suggested it.